docs/api_requests/patch-block.md

```diff
@@ -98,9 +100,41 @@ operation can be undone if the guest issues a write for the last sector of the
 raw block device, or the guest application can become inconsistent or/and can
 create inconsistency in the block device itself.
 
-### In flight I/O requests
+#### In flight I/O requests
 
 If the atomicity of the operation is guaranteed by using methods to make the
 microVM quiescence during the update sequence (for example pausing the microVM)
 the guest itself or block device can still become incosistent from in flight
 I/O requests in the guest that will be executed after it is resumed.
+
+## Updating vhost-user block devices after boot
+
+Unlike with Virtio block device, with vhost-user block devices, Firecracker
+does not interact with the underlying block file directly (the vhost-user
+backend does). It means that changes to the file are not automatically seen
+by Firecracker. There is a mechanism in the
+[vhost-user protocol](https://qemu-project.gitlab.io/qemu/interop/vhost-user.html)
+for the backend to notify the frontend about changes in the device config
+via `VHOST_USER_BACKEND_CONFIG_CHANGE_MSG` message. This requires an extra
+UDS socket connection between the frontend and backend used for
+backend-originated messages. This mechanism **is not supported**
+by Firecracker. Instead, Firecracker makes use of the `PATCH /drives`
+API request to get notified about such changes. Such an API request
+only includes the required property (`drive_id`), because optional properties
+are not relevant to vhost-user.
```

Comment:
Should we mention PATCH here, considering it is not implemented yet? I think we can add this note in the PR that adds config notification. Plus it would be nice to add it to the vhost-user set of PRs in the CHANGELOG.

Reply:
I have already mentioned the config change PR in the changelog. I don't think it's worth delaying this doc part, because we already know we must merge that other PR.